pkg/kubecost/allocation.go

```diff
@@ -337,9 +372,12 @@ func (a *Allocation) MarshalJSON() ([]byte, error) {
 	jsonEncodeFloat64(buffer, "cpuCoreUsageAverage", a.CPUCoreUsageAverage, ",")
 	jsonEncodeFloat64(buffer, "cpuCoreHours", a.CPUCoreHours, ",")
 	jsonEncodeFloat64(buffer, "cpuCost", a.CPUCost, ",")
```

Comment:
I was thinking that they should be sent separately because that is how it is done with asset adjustments such as node. I think you are right though, because in that case adjustment is its own column in the UI, That isn't going to work here though since there will be 5+ adjustments. So I think you are probably right.

Reply:
Okay, great. It's also different for Assets (e.g. `Node`) because there's just one `Adjustment` that adjusts `TotalCost`, and `TotalCost` is send with adjustment-included. We don't (AFAIK) send unadjusted costs when it's possible to send the complete cost.

Reply:
This does break the marshal/unmarshal equality test however, so maybe it is fine leaving it up to the front end to do the simple addition on these values or a total field could be added for each cost type to hold the result of the function

Reply:
For posterity: we discussed in Slack, and @Sean-Holcomb has convinced me that the way is stands is good.